requirements/base.txt

```diff
@@ -30,3 +31,10 @@ google-api-python-client==1.7.12
 verto==0.10.0
 python-markdown-math==0.6
 PyYAML==5.3.1
+
+# Style checker
+# Replace flake8 dependency once new release is created
```

Comment:
Might need to make a note somewhere else or an issue to do this so we don't forget about it after it's merged